src/xeus_calc_interpreter.cpp

```diff
@@ -130,19 +130,27 @@ namespace xeus_calc
                 publish_stream("stdout", "Operate\t\t");
                 double secondOperand = stack.back();
                 stack.pop_back();
-                double firstOperand = stack.back();
-                stack.pop_back();
-                if (token == "*")
-                    stack.push_back(firstOperand * secondOperand);
-                else if (token == "/")
-                    stack.push_back(firstOperand / secondOperand);
-                else if (token == "-")
-                    stack.push_back(firstOperand - secondOperand);
-                else if (token == "+")
-                    stack.push_back(firstOperand + secondOperand);
-                else if (token == "^")
-                    stack.push_back(std::pow(firstOperand, secondOperand));
-                /*else
+                if (stack.empty())
```

Comment:
Does it prevent having an error when doing `+`? I feel like the `pop_back` on line 132 would make the kernel crash